src/main/java/org/ngafid/accounts/UserPreferencesEmails.java

```diff
@@ -0,0 +1,131 @@
+package org.ngafid.accounts;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.ngafid.Database;
+import org.ngafid.EmailType;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+import java.util.HashMap;
+
+import java.util.logging.Logger;
+
+import static org.ngafid.flights.calculations.Parameters.*;
+
+
+
+public class UserPreferencesEmails {
+
+    private int userId;
+    private HashMap<String, Boolean> emailTypesUser;
+    private String[] emailTypesKeys;
+
+    private static HashMap<Integer, User> users = new HashMap<Integer, User>();
+    private static HashMap<String, HashMap<String, Boolean>> emailTypesUsers = new HashMap<String, HashMap<String, Boolean>>();
```

Comment:
I don't know if we should cache these things since they could be changed at any time by the web server?